Lib/inspect.py

```diff
@@ -3179,6 +3179,8 @@ def _bind(self, args, kwargs, *, partial=False):
         parameters_ex = ()
         arg_vals = iter(args)
 
+        pos_only_param_in_kwargs = []
```

Comment:
I'm taking a look and see whether this can ever truly have more than one item given the current control flow.

Reply:
Fixed in 69162908a945b998b62a39af9089c6feb8e6b4ff